android/libraries/rib-android/gradle.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2017. Uber Technologies
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+POM_NAME=RIBs (Android)
+POM_ARTIFACT_ID=rib-android
+POM_PACKAGING=android
```

Comment:
Just curious, what is resulting package name?

Reply:
Group is `GROUP=com.badoo.ribs`
this option is not about package name, it's to specify that it's an android module :)

Reply:
Yeah, I just did not notice the group name, so was confused a bit =)